gtfs_realtime_translators/factories/factories.py

```diff
@@ -110,3 +110,10 @@ def create(*args, **kwargs):
         message = gtfs_realtime.FeedMessage(header=header,
                                             entity=entities)
         return message
+
+
+class RequiredFieldValidator:
+    @staticmethod
+    def validate(fieldName, fieldValue):
+        if fieldValue is None:
+                raise ValueError('{} is required.'.format(fieldName))
```

Comment:
RequiredFieldValidator is not a "factory", so I would move this to another location. 

I would just add a `validate(self, field_name, field_value)` method to the translator classes that need it for now. In the future, we should make a base class from which each org translators inherit a single method

Reply:
Agree, will move it appropriately.